src/main/java/mods/eln/Eln.java

```diff
@@ -4955,7 +5109,7 @@ private void registerMiscItem(int id) {
         {
             subId = 23;
             GenericItemUsingDamageDescriptorWithComment desc = new GenericItemUsingDamageDescriptorWithComment(
-                TR_NAME(Type.NONE, "Advanced Machine Block"), new String[]{}); // TODO: Description.
+                TR_NAME(Type.NONE, "Advanced Machine Block"), new String[]{});
```

Comment:
Don't remove TODOs without doing what they say, when what they say to do is quite reasonable. Just add the description. :P